### PR TITLE
#165377719 Query All Room Resources by RoomId

### PR DESCRIPTION
**Pull request by verenceLola**
### What does this PR do?
Adds query for getting all resources assigned to a given room
### Description of the task to be completed?
As an admin, it should be possible to get all the resources assigned to a given room by suppliying the query with the desired `roomId`
 - Clone the branch and follow the following steps to [setup](https://github.com/andela/mrm_api).
 - Checkout by running `git checkout ft-query-all-resources-165377719`
 - Follow instructions to assign a resource to a room from #353 
 - Run the following query as an **admin** to fetch all the resources assigned to a given `roomId`
```
{
  getResourcesByRoomId(roomId:1){
    roomResources{
      name
      id
      quantity
    }
  }
}
```
### Any background context you want to provide?
N/A
### What are the relevant pivotal tracker stories?
[#165377719](https://www.pivotaltracker.com/story/show/165377719)
### Screenshots
 - #### Existing room resources:
image

 - #### Non existing `roomId`:
image

 - #### Get room resources by a `non admin`:
image

### CODE REVIEW

File: api/room_resource/schema.py [hunk 1]
```diff
@@ -197,16 +197,30 @@ def resolve_all_resources(self, info, **kwargs):
         resp = PaginatedResource(**kwargs)
         return resp
 
+    @Auth.user_roles('Admin')
     def resolve_get_resources_by_room_id(self, info, room_id):
         # Get resources of a specifics room
```
Comment (TeamoreA) [inline api/room_resource/schema.py]: Kindly correct this from `specifics` to `specific` since it involves the method you are using.

**Comment by verenceLola** [comment]
> Good work @verenceLola However, kindly update the query in the PR description to
> 
> ```
>  query{
>   getResourcesByRoomId(roomId:1){
>       name
>       id
>       quantity
>     }
>   }
> ```
> The current one does not work

Kindly pull the new changes and review. It should be working okay now. Thanks for the feedback though @kwanj-k

File: tests/base.py [hunk 2]
```diff
@@ -98,6 +99,11 @@ def setUp(self):
             resource = Resource(name='Markers',
                                 quantity=3)
             resource.save()
+            resource_2 = Resource(
+                name="Pencils",
+                quantity=5
+            )
+            resource_2.save()
```
Comment (rajeman) [inline tests/base.py]: @verenceLola is there any reason why you saved another resource to the DB instead of using the one above it?

Comment (rajeman) [reply tests/base.py]: Thanks. I got the explanation during the sync.

**Comment by verenceLola** [comment]
> Fix the failing tests

This has been fixed. @Bonifase

File: api/room_resource/schema.py (hunk 1 above)
Comment (verenceLola) [reply api/room_resource/schema.py]: This has been fixed. Thanks for the feedback @TeamoreA

**Pull request merged**